docs/extend/modular-resources/configure.md

```diff
@@ -1,30 +1,171 @@
 ---
-title: "Configure your module and modular resource"
+title: "Configure a module on your robot"
 linkTitle: "Configure"
 weight: 30
 type: "docs"
 tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
-description: "Use the Viam module system to implement custom resources that can be included in any Viam-powered robot."
+description: "Configure a module on your robot."
 no_list: true
 ---
 
-Configure a module on your robot to make one or more modular resources available for configuration.
+Add and configure a module on your robot to make one or more modular resources available for configuration.
+You can [add a module from the Viam Registry](#add-a-module-from-the-viam-registry), or you can [code your own module and add it to your robot locally](#add-a-local-module-to-your-robot).
 
-If you are developing your own modular resource, you must first follow [these steps](/extend/modular-resources/create/) to code your own module and generate an executable.
-If you are using a pre-built modular resource, make sure you install the module and determine the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
+{{% alert title="Modules vs. modular resources" color="tip" %}}
 
-Follow these steps to configure the module and modular resource:
+A configured *module* can make one or more *modular resources* available for configuration.
```

Comment:
Also if this is still the case, and this tip stays here, it's confusing because it seems the user doesn't actually have to explicitly configure a module from the registry, but rather just a modular resource made available by that module, which implicitly autoconfigures the module??

Reply:
Somewhat. Both module and resource-provided-by module are configurable, this page for both. Adding the module-provided resource does implicitly auto-add the module, but with default settings (like pin to patch). You may wish to also-configure module beyond resource-provided-by-module.

Inlined guidance (removed as-tip, now as-paragraph)

This copy also updated in [the other PR](https://github.com/viamrobotics/docs/pull/1705) with further clarifications.